### ISSUE TITLE
Repeatedly logging out

### ISSUE THREAD

**Issue by tajbinshawon** [nextcloud/android#5438]
************ CAUSE OF ERROR ************

java.net.SocketTimeoutException: timeout
	at okhttp3.internal.http2.Http2Stream$StreamTimeout.newTimeoutException(Http2Stream.java:671)
	at okhttp3.internal.http2.Http2Stream$StreamTimeout.exitAndThrowIfTimedOut(Http2Stream.java:679)
	at okhttp3.internal.http2.Http2Stream.takeHeaders(Http2Stream.java:153)
	at okhttp3.internal.http2.Http2Codec.readResponseHeaders(Http2Codec.java:131)
	at okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.java:88)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:45)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:127)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:257)
	at okhttp3.RealCall.execute(RealCall.java:93)
	at com.nextcloud.common.OkHttpMethodBase.execute(OkHttpMethodBase.kt:126)
	at com.nextcloud.common.NextcloudClient.execute(NextcloudClient.kt:84)
	at com.owncloud.android.lib.resources.activities.GetActivitiesRemoteOperation.run(GetActivitiesRemoteOperation.java:131)
	at com.nextcloud.common.NextcloudClient.execute(NextcloudClient.kt:80)
	at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment.lambda$fetchAndSetData$5$FileDetailActivitiesFragment(FileDetailActivitiesFragment.java:331)
	at com.owncloud.android.ui.fragment.-$$Lambda$FileDetailActivitiesFragment$N7POe6TZvoGcuV2qP-niPxY_7t4.run(Unknown Source:12)
	at java.lang.Thread.run(Thread.java:764)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30100190
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: xiaomi
Device: tulip
Model: Redmi Note 6 Pro
Id: PKQ1.180904.001
Product: tulip

************ FIRMWARE ************
SDK: 28
Release: 9
Incremental: V11.0.2.0.PEKMIXM

**Comment by ezaquarii** [comment]
Fixed in https://github.com/nextcloud/android-library/pull/394

**Comment by tobiasKaminsky** [comment]
Can you try out our dev version (you can install it in parallel) and give us feedback:
https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#dev-release

**Comment by AndyScherzinger** [comment]
Duplicate of #5331

**Comment by tobiasKaminsky** [comment]
@tajbinshawon is this issue still happening for you with latest dev version?

**Comment by tobiasKaminsky** [comment]
3.11 RC1 is out, can you please test this via gplay beta program?

**Comment by tobiasKaminsky** [comment]
No response, please re-open if if still happens.
